js/subsites-menu.js

```diff
@@ -11,10 +11,8 @@
 
       const { mobileBreakpointJS } = drupalSettings.localgov_base;
 
-      const subsitesMenuToggle = document.querySelector(
-        '.subsite-extra__header-toggle-button',
-      );
-      const subsitesMenu = document.querySelector('.subsite-extra-menu');
+      const [subsitesMenuToggle] = once('allSubsitesMenuToggles', '.subsite-extra__header-toggle-button', context);
+      const [subsitesMenu] = once('allSubsitesMenuToggles', '.subsite-extra-menu', context);
```

Comment:
```suggestion
      const [subsitesMenu] = once('allSubsitesMenuToggles', '.subsite-extra-menu', context);
      
      if (!subsitesMenuToggle || !subsitesMenu) {
        return;
      }
```

This is untested, but it's the usual defensive companion to `.once()`.

Reply:
Thanks @ctorgalson, I've added in the extra defensive clause now. Let me know if you think we're missing any thinbg.